keyboards/handwired/ndk/ndk16/ndk16.c

```diff
@@ -0,0 +1,200 @@
+        0x60, 0x10, 0x10, 0x10, 0x10, 0x10, 0x08, 0x04, 0x02, 0x79, 0x7d, 0x02, 0xfc, 0x00, 0x00, 0x00, 
+        0x00, 0x00, 0x00, 0x00, 0x03, 0x0c, 0x33, 0xc7, 0x8f, 0x9f, 0x87, 0xc1, 0xe0, 0x20, 0x04, 0x04, 
+        0x0e, 0x8a, 0x84, 0x80, 0x00, 0x00, 0x00, 0x20, 0x22, 0x85, 0x82, 0x00, 0x07, 0x08, 0x10, 0xe0, 
+        0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x8f, 0x7f, 0x7f, 0x67, 0x30, 0x00, 0x00, 0x00, 
+        0x00, 0x81, 0xc0, 0xc0, 0xe0, 0xe0, 0xe0, 0xe1, 0xe1, 0xf0, 0xf0, 0x08, 0xe4, 0x10, 0x0e, 0x01, 
+        0x00, 0x00, 0x00, 0x00, 0x80, 0x60, 0x18, 0x06, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x3c, 0xfe, 
+        0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xfe, 0xc1, 0x3e, 0xc0, 0x00, 
+        0x00, 0xf0, 0x0c, 0x02, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+        0x03, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xf0, 0xff, 0x00, 
+        0x07, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+        0x00, 0x07, 0x07, 0x07, 0x07, 0x07, 0x07, 0x07, 0x07, 0x07, 0x07, 0x07, 0x07, 0x07, 0x07, 0x00
+    };
+
+    oled_write_raw_P(raw_cat, sizeof(raw_cat));
+}
+
+/* cat yawn mouth image; image from https://github.com/codeyfrommars/Gamepad */
+static void render_cat_yawn(void) {
+    static const char PROGMEM raw_cat_yawn[] = {
+        0x00, 0x38, 0xe4, 0x04, 0xe4, 0xe4, 0xe8, 0xc8, 0x10, 0x20, 0x20, 0x40, 0x40, 0x40, 0x20, 0x20, 
+        0x10, 0x10, 0x10, 0x10, 0x10, 0x18, 0x1c, 0x1a, 0x45, 0xc1, 0x0e, 0xf0, 0x00, 0x00, 0x00, 0x00, 
+        0x00, 0x00, 0x03, 0x0e, 0x11, 0xe7, 0x8f, 0x9f, 0x9f, 0xc0, 0x00, 0x00, 0x00, 0x02, 0x83, 0xe1, 
+        0xf1, 0xf0, 0xf8, 0xfc, 0xfd, 0xfd, 0xfc, 0xfc, 0xfc, 0xfc, 0xf8, 0xf0, 0xe1, 0xc2, 0x0c, 0xf0, 
+        0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x0f, 0xff, 0x39, 0x00, 0x00, 0x00, 0x00, 0x0f, 0x1f, 
+        0x3f, 0x3f, 0x7f, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0x7f, 0x7f, 0x3f, 0x0f, 0x83, 0x70, 0x0f, 
+        0x00, 0x00, 0x00, 0x80, 0x60, 0x18, 0x06, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x0e, 0x1e, 
+        0xfc, 0xfc, 0xfc, 0xfc, 0xfc, 0xfc, 0xfc, 0xfc, 0xfc, 0xfc, 0x82, 0x7e, 0xc1, 0x00, 0x00, 0x00, 
+        0xc0, 0x38, 0x06, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+        0x1f, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xfe, 0xc1, 0xfe, 0x00, 0x00, 
+        0x07, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+        0x00, 0x07, 0x07, 0x07, 0x07, 0x07, 0x07, 0x07, 0x07, 0x07, 0x07, 0x07, 0x07, 0x07, 0x00, 0x00
+    };
+
+    oled_write_raw_P(raw_cat_yawn, sizeof(raw_cat_yawn));
+}
+
+
+#if defined(OLED_ENABLE)
+
+/* oled fix for 'cold boot' */
+void keyboard_post_init_user(void) {
+    wait_ms(2000);
+    oled_init(OLED_ROTATION_0);
+}
```

Comment:
This shouldn't be needed, and is odd that it is.

Also, function should be the `kb` version, not the user.